easydata.net/src/EasyData.Core/Services/EasyDataManager.cs

```diff
@@ -46,28 +46,108 @@ public EasyDataManager(IServiceProvider services, EasyDataOptions options)
 
         public async Task<MetaData> GetModelAsync(string modelId, CancellationToken ct = default)
         {
-            if (Model.Id !=modelId) {
+            if (Model.Id != modelId)
+            {
                 //TODO: Try to load model from cache
 
                 await LoadModelAsync(modelId, ct);
             }
 
+            UpdateModelMetaWithOptions();

```

Comment:
Why do we need to update the model from options every time user makes a request?